api/projectTeam.js

```diff
@@ -23,13 +27,13 @@ router.get("/self", async (req, res) => {
 router.put("/join/:name", async (req, res) => {
   try {
     const projectTeam = await models.ProjectTeam.findOne({
-      where: { name: req.params.name },
-      include: [models.Person]
+      where: { name: req.params.name }
     });
-    if (projectTeam.People.length >= 4) {
+    if (projectTeam.Members.length >= 4) {
       throw new Error("Team is full");
```

Comment:
Wait, if this is a route, can we just `return res.status(400).json({ /* error msg */})`